### ISSUE TITLE
Private Window does not say 'Private' in UK English - follow up to 6872

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#6890]
## Description 
Found while testing #6872 

If you open a Private window while using UK English, the window doesn't say 'Private' any more.

## Steps to Reproduce

   1. Using UK english, launch clean profile in a terminal
   2. Open Private Window
   3. 

## Actual result:
<img width="1241" alt="Screen Shot 2019-11-12 at 10 15 05 AM" src="https://user-images.githubusercontent.com/28145373/68684056-d436fb00-0535-11ea-860b-3fa86f060fe3.png">

Terminal errors:
```
[3184:3184:1112/181859.940022:ERROR:message_formatter.cc(91)] MessageFormat({0, plural,
      =1 {Private}
      other {{Private (#)}
    }) failed with U_PATTERN_SYNTAX_ERROR
[3184:3184:1112/181859.940473:ERROR:message_formatter.cc(91)] MessageFormat({0, plural,
      =1 {Private}
      other {{Private (#)}
    }) failed with U_PATTERN_SYNTAX_ERROR
[3184:3184:1112/181859.940575:ERROR:message_formatter.cc(91)] MessageFormat({0, plural,
      =1 {Private}
      other {{Private (#)}
    }) failed with U_PATTERN_SYNTAX_ERROR
[3184:3184:1112/181859.955558:ERROR:message_formatter.cc(91)] MessageFormat({0, plural,
      =1 {Private}
      other {{Private (#)}
    }) failed with U_PATTERN_SYNTAX_ERROR

```

## Expected result:
No terminal errors
<img width="1241" alt="Screen Shot 2019-11-12 at 10 16 23 AM" src="https://user-images.githubusercontent.com/28145373/68684069-d8fbaf00-0535-11ea-9c15-050a6cf7f2f7.png">

## Reproduces how often: 
easily

## Brave version (brave://version info)

Brave | 0.71.115 Chromium: 78.0.3904.97 (Official Build) (64-bit)
-- | --
Revision | 021b9028c246d820be17a10e5b393ee90f41375e-refs/branch-heads/3904@{#859}
OS | macOS Version 10.13.6 (Build 17G5019)

## Version/Channel Information:

- Can you reproduce this issue with the current release?
- Can you reproduce this issue with the beta channel?
- Can you reproduce this issue with the dev channel?
- Can you reproduce this issue with the nightly channel?

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:
cc @petemill

**Comment by kjozwiak** [comment]
@rebron @bsclifton moving this into the `0.71.x` milestone as we'll probably want this fixed before we release tomorrow. It's a regression and might confuse new users who aren't as familiar with private windows/mode.

**Comment by btlechowski** [comment]
Reproduced on 

Brave | 0.71.115 Chromium: 78.0.3904.97 (Official Build) (64-bit)
-- | --
Revision | 021b9028c246d820be17a10e5b393ee90f41375e-refs/branch-heads/3904@{#859}
OS | Ubuntu 18.04 LTS

![image](https://user-images.githubusercontent.com/34715963/68692227-67dbec00-0575-11ea-92c3-928d1b41c503.png)

**Comment by GeetaSarvadnya** [comment]
Reproduced in Windows 10 x64- 0.71.115

![image](https://user-images.githubusercontent.com/38657976/68694541-5a3b5c00-059f-11ea-85aa-963b703ffc59.png)

**Comment by bsclifton** [comment]
Fixed in 0.71 with https://github.com/brave/brave-core/pull/3982

**Comment by LaurenWags** [comment]
Verified passed with

Brave | 0.71.116 Chromium: 78.0.3904.97 (Official Build) (64-bit)
-- | --
Revision | 021b9028c246d820be17a10e5b393ee90f41375e-refs/branch-heads/3904@{#859}
OS | macOS Version 10.13.6 (Build 17G5019)

* Verified STR from description.
* No terminal errors were observed.
<img width="1278" alt="Screen Shot 2019-11-12 at 5 58 43 PM" src="https://user-images.githubusercontent.com/28145373/68718093-4fb89c80-0576-11ea-853f-aed666f4a297.png">
